spec/models/user_spec.rb

```diff
@@ -481,4 +481,39 @@
       end
     end
   end
+
+  describe "#status" do
+    context "when the user is active" do
+      let(:user) { create(:user, last_sign_in_at: Time.zone.yesterday) }
+
+      it "returns the status of the user" do
+        expect(user.status).to eq(:active)
+      end
+    end
+
+    context "when the user is not signed in" do
```

Comment:
has not?

Reply:
Does it make much difference? I think we use is not in the rest of the codebase (?)

Reply:
This test is specifically about when the user has never signed in though, rather than whether they currently are.